migrate/livesync-for-s3.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Livesync S3 to Timescale Cloud
+excerpt: Synchronize data from S3 to Timescale Cloud service in real time
+products: [cloud]
+keywords: [migration, low-downtime, backup]
+tags: [recovery, logical backup, replication]
+---
+
+import PrereqCloud from "versionContent/_partials/_prereqs-cloud-only.mdx";
+import EarlyAccessNoRelease from "versionContent/_partials/_early_access.mdx";
+
+# Livesync from S3 to Timescale Cloud
+
+You use $LIVESYNC to synchronize all the data, or specific tables, from an S3 bucket to your
+$SERVICE_LONG in real-time. You run $LIVESYNC continuously, turning S3 into a primary database with your
+$SERVICE_LONG as a logical replica. This enables you to leverage $CLOUD_LONG’s real-time analytics capabilities on
+your replica data.
+
```

Comment:
Maybe a diagram here to illustrate how we envision this feature? 

![image](https://github.com/user-attachments/assets/3533df8d-79c3-4e80-be9b-bbc7868b5ecb)

Reply:
I'd like to look at this for another iteration of the doc. Need to have design give us a beautiful image.